tests/Zend/Controller/_files/Plugins/ThrowingPlugin.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace MyApp\Controller\Plugin;
+use stdClass;
+use Zend_Controller_Request_Abstract;
+
+require_once 'Zend/Controller/Action/Helper/Abstract.php';
+class ThrowingPlugin extends \Zend_Controller_Plugin_Abstract {
+
+    public function dispatchLoopStartup(Zend_Controller_Request_Abstract $request)
+    {
+        $this->produceTypeError();
+        throw new \Exception('Should not ever get here');
+    }
+
+    private function produceTypeError(): self
```

Comment:
These tests would fail on older versions of php right?

Reply:
Yes, especially on versions that don't have return types (i.e. <7.0). Maybe they could be mark as skipped based on PHP version... what do you think?

Reply:
Now i think about it, the rest of the code would also fail on php < 7.0

I havent tried it, but i read that this is the backwards compatible way to do it:
```
try {
    // Code that may throw an Exception or Error.
} catch (Throwable $t) {
    // Executed only in PHP 7, will not match in PHP 5.x
} catch (Exception $e) {
    // Executed only in PHP 5.x, will not be reached in PHP 7
}
```

Reply:
Don't worry about it, my co-worker who I gave permissions in this repo introduced throwable catches into the codebase a couple of weeks ago to solve some type error issues, so I've gone and raised the required php version to 7.0

Reply:
too bad... we still have some systems running on php 5.6
i'll send a pull request to make it compatible again ok?

Reply:
Yep that's absolutely fine, I've actually set you as a collaborator now so feel free to merge stuff like that as you see fit. You'll have to change the composer.json value back